airflow/ui/src/components/SearchBar.tsx

```diff
@@ -0,0 +1,59 @@
+/*!
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+import {
+  Button,
+  type ButtonProps,
+  Input,
+  InputGroup,
+  type InputGroupProps,
+  InputLeftElement,
+  type InputProps,
+  InputRightElement,
+} from "@chakra-ui/react";
+import { FiSearch } from "react-icons/fi";
+
+export const SearchBar = ({
```

Comment:
This was moved out from `DagList` to satisfy the rule to have one component per file.